Documentation/Storage-Configuration/Monitoring/ceph-monitoring.md

```diff
@@ -21,7 +21,7 @@ First the Prometheus operator needs to be started in the cluster so it can watch
 A full explanation can be found in the [Prometheus operator repository on GitHub](https://github.com/prometheus-operator/prometheus-operator), but the quick instructions can be found here:
 
 ```console
-kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/v0.40.0/bundle.yaml
+kubectl create -f https://raw.githubusercontent.com/coreos/prometheus-operator/v0.71.1/bundle.yaml
```

Comment:
Usually, we prefer to use `kubectl apply` because it doesn't fail if the resource already exists. Is there a particular reason why this is changed to `create`?

Reply:
kubectl apply command causing following error: 
`metadata.annotations: Too long: must have at most 262144 bytes`
More information is [here](https://github.com/prometheus-operator/prometheus-operator/issues/4355)

Reply:
What is the solution if prometheus was previously installed? As @richardelling indicated, `create` will fail in that case. From Rook's perspective maybe it's fine for this doc, but perhaps it would help users if we add a note about what to do for upgrades, even if we just link to some prometheus topic.

Reply:
I have written a gist where it explains what is the issue and how to do a workaround.

Reply:
Would you mind either opening a Rook issue, or commenting on #13459 with those same contents in your gist? We just don't typically link to external topics like that. Thanks!

Reply:
I opened an issue and replaced gist link with the new issue.  issue is https://github.com/rook/rook/issues/13826

Reply:
I commented the explanation on the issue, and linked it in a warning note